.zenodo.json

```diff
@@ -0,0 +1,132 @@
+{
+    "title": "Plots.jl",
+    "license": "MIT", 
+    "creators": [
+        {
+            "affiliation": "Elemental Cognition", 
+            "name": "Tom Breloff"
+        }
+    ],
+    "contributors":[
+        {
+            "affiliation": "TU Wien", 
+            "name": "Daniel Schwabeneder",
+            "orcid": "0000-0002-0412-0777",
+            "type": "ProjectLeader"
+        }, 
+        {
+            "affiliation": "GLOBE Institute", 
+            "name": "Michael Krabbe Borregaard",
+            "type": "ProjectLeader"
+        }, 
+        {
+            "affiliation": "Leibniz Universit\u00e4t Hannover", 
+            "name": "Simon Christ",
+            "orcid": "0000-0002-5866-1472",
+            "type": "ProjectMember"
```

Comment:
```suggestion
            "type": "ProjectLeader"
```

Reply:
I did the classification based on the share of the current codebase, but I am also fine with your suggestions